README.md

```diff
@@ -183,22 +152,14 @@ You will use play-testing and unit tests to drive your development.
 
 At submission time, no matter where you are, submit the project via Learn.
 
-In addition to submitting your Pull Request, submit a reflection the the pair programming reflection prompt.
-
 
 ## Project Directions
 
-### Setup a Pair Plan
-
-First, come up with a "plan of action" for how you want to work as a pair. Discuss your learning style, how you prefer to receive feedback, and one team communication skill you want to improve with this experience.
-
-We recommend spending at least a portion of the time pair programming and working collaboratively from the same machine.  Zoom and screen sharing or [VSCode Live Share](https://code.visualstudio.com/learn/collaboration/live-share) are good tools to consider.  Some teams will choose to pair program and work collaboratively from the same machine for the entire project.  Some teams will choose to divide a portion of the work and combine their code using git. 
-
 ### Get Familiar
 
-Take time to read through the Wave 1 implementation requirements and the tests for wave 1. Write down your questions, and spend some time going through your understanding of the requirements and tests with your pair. Make sure you both can run `$ pytest` and see the tests fail.
+Take time to read through the Wave 1 implementation requirements and the tests for wave 1. Write down your questions, and spend some time going through your understanding of the requirements. Make sure you can run `$ pytest` and see the tests fail.
 
-If, after you and your pair have taken some time to think through the problem and would like direction for how to dissect the problem, or if you need clarity on the terms/vocabulary we used in this project, you can check out [a small hint we've provided](./project_docs/hints.md).
+If, after you have taken some time to think through the problem and would like direction for how to dissect the problem, or if you need clarity on the terms/vocabulary we used in this project, you can check out [a small hint we've provided](./project_docs/hints.md).
```

Comment:
Can't drop a comment on it since it's outside the diff, but the last paragraph of wave 1 has a sentence "It is worth spending some time with your partner to think carefully about this." that we likely want to update.